api/pkg/apis/v1alpha1/providers/stage/create/create.go

```diff
@@ -198,6 +199,24 @@ func (i *CreateStageProvider) Process(ctx context.Context, mgrContext contexts.M
 				mLog.ErrorfCtx(ctx, "  P (Create Stage) process failed, failed to delete instance: %+v", err)
 				return nil, false, err
 			}
+			for ic := 0; ic < i.Config.WaitCount; ic++ {
+				remainings := utils.FilterIncompleteDelete(ctx, &i.ApiClient, objectNamespace, []string{objectName}, true, i.Config.User, i.Config.Password)
+				if len(remainings) == 0 {
+					return outputs, false, nil
+				}
+				mLog.InfofCtx(ctx, "  P (Create Stage) process: Waiting for instance deletion: %+v", remainings)
+				time.Sleep(time.Duration(i.Config.WaitInterval) * time.Second)
+			}
+			providerOperationMetrics.ProviderOperationErrors(
+				create,
+				functionName,
+				metrics.ProcessOperation,
+				metrics.RunOperationType,
+				v1alpha2.DeploymentNotReached.String(),
+			)
+			err = v1alpha2.NewCOAError(nil, "Instance deletion reconcile timeout", v1alpha2.InternalError)
```

Comment:
We can write the remaining count in error message.

Reply:
There is only one instance per create stage provider, either in create or remove case.